doc/userguide/rules/http-keywords.rst

```diff
@@ -843,4 +843,16 @@ Notes
 Multiple Buffer Matching
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-``file.data`` supports multiple buffer matching, see :doc:`multi-buffer-matching`.
\ No newline at end of file
+``file.data`` supports multiple buffer matching, see :doc:`multi-buffer-matching`.
+
+file.name
+---------
+
+The ``file.name`` keyword can be used at the HTTP application level.
+
+Example::
+
+  alert http any any -> any any (msg:"http layer file.name keyword usage"; \
+  file.name; content:"picture.jpg"; classtype:bad-unknown; sid:1; rev:1;)
```

Comment:
Any special reason not to use the `example-rule` container here?

Reply:
Hm, it looks like I missed updating that one. I can either submit a new PR or my plan is to have a larger update to http-keywords.rst based on our conversation around the keyword table and other updates around https://redmine.openinfosecfoundation.org/issues/3025 in the next couple days

Reply:
that second option sounds good to me :)